content/articles/little-printer.md

```diff
@@ -0,0 +1,112 @@
+Title: Berg's Little Printer
+Date: 2015-04-30
+Category: Misc
+Authors: Tom
+Short_summary: We did some pretty exciting work for Berg!
+
+*Reading time: ~5 minutes.*
+
+I'm a big fan of the amazing London design agency Berg. So when its
+CEO Matt Berg tweeted this last year:
+
+<blockquote class="twitter-tweet" lang="en"><p lang="en" dir="ltr">I&#39;m after a London Python freelancer for a 2mo gig, Flask/tests/Redis/Bootstrap. Mail me... matt at interconnected dot org. Tell yr friends!</p>&mdash; Matt Webb (@genmon) <a href="https://twitter.com/genmon/status/537304016993386497">November 25, 2014</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+I was hooked.
+
+<!-- PELICAN_END_SUMMARY -->
+
+We emailed Matt and it turned out that he was looking for someone to
+re-implement a server for their
```

Comment:
I think the article is good, but I'd be interested to know what kind of things this server does. (Print things remotely, register printers ..)